osbs/cli/main.py

```diff
@@ -349,6 +356,17 @@ def cmd_restore(args, osbs):
     logger.info("backup recovery complete!")
 
 
+def cmd_oauth_token(args, osbs):
+    uri = urljoin(osbs.os_conf.get_openshift_base_uri(), "/token/request")
```

Comment:
Indeed, gonna fix that, although my test instance somehow managed to redirect correctly